tests/php/src/BlockUniqidTransformerTest.php

```diff
@@ -0,0 +1,154 @@
+				'style_handle'           => 'wp-duotone-621e12fb51e3a',
+				'style_content'          => ".wp-duotone-621e12fb51e3a > .wp-block-cover__image-background, .wp-duotone-621e12fb51e3a > .wp-block-cover__video-background{filter:url(\'#wp-duotone-621e12fb51e3a\') !important;}",
+				'expected_style_content' => ".wp-duotone-1 > .wp-block-cover__image-background, .wp-duotone-1 > .wp-block-cover__video-background{filter:url(\'#wp-duotone-1\') !important;}",
+			],
+			'transform_container_class_name'         => [
+				'block_content'          => '<div class="wp-container-621e133aaf0e2 wp-block-group is-style-default has-black-background-color has-background" style="padding-top:20px;padding-right:20px;padding-bottom:20px;padding-left:20px">This is a super cool class name: <code>wp-container-0123456789abc</code>!</div>',
+				'expected_block_content' => '<div class="wp-container-2 wp-block-group is-style-default has-black-background-color has-background" style="padding-top:20px;padding-right:20px;padding-bottom:20px;padding-left:20px">This is a super cool class name: <code>wp-container-0123456789abc</code>!</div>',
+				'style_handle'           => null,
+				'style_content'          => null,
+				'expected_style_content' => null,
+			],
+			'ignore_class_names_without_hash'        => [
+				'block_content'          => '<div class="wp-duotone-test wp-block-cover">This is a super cool class name: <code>wp-duotone-0123456789abc</code>!</div>',
+				'expected_block_content' => '<div class="wp-duotone-test wp-block-cover">This is a super cool class name: <code>wp-duotone-0123456789abc</code>!</div>',
+				'style_handle'           => null,
+				'style_content'          => null,
+				'expected_style_content' => null,
+			],
+			'ignore_already_transformed_class_names' => [
+				'block_content'          => '<div class="wp-duotone-1 wp-block-cover">This is a super cool class name: <code>wp-duotone-0123456789abc</code>!</div>',
+				'expected_block_content' => '<div class="wp-duotone-1 wp-block-cover">This is a super cool class name: <code>wp-duotone-0123456789abc</code>!</div>',
+				'style_handle'           => null,
+				'style_content'          => null,
+				'expected_style_content' => null,
+			],
+		];
+	}
+
+	/**
+	 * @covers ::transform_class_names_in_block_content()
+	 * @covers ::transform_class_names_in_inline_styles()
+	 * @covers ::get_class_name_regexp_pattern()
+	 * @covers ::unique_id()
+	 *
+	 * @dataProvider get_block_data
+	 *
+	 * @param string      $block_content
+	 * @param string      $expected_block_content
+	 * @param string|null $style_handle
+	 * @param string|null $style_content
+	 * @param string|null $expected_style_content
+	 */
+	public function test_transform_class_names( $block_content, $expected_block_content, $style_handle, $style_content, $expected_style_content ) {
+		$this->markTestIncomplete( 'Test neds to migrate over to tests for AMP_Block_Uniqid_Sanitizer.' );
```

Comment:
I didn't port the tests over to a sanitizer test class.

Reply:
I've moved them over in 5c7cffe.